models/intermediate/glific/glific_messages_deserialize.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(materialized="table", schema="dalgo_intermediate") }}
```

Comment:
You don't need a schema here. Can you copy the macro from Sneha dbt which automatically does this?

Reply:
I dont want the schema name as `intermediate`